server/channels/api4/outgoing_oauth_connection.go

```diff
@@ -63,7 +63,7 @@ func ensureOutgoingOAuthConnectionInterface(c *Context, where string) (einterfac
 		return nil, false
 	}
 
-	if c.App.OutgoingOAuthConnections() == nil || c.App.License() == nil || c.App.License().SkuShortName != model.LicenseShortSkuEnterprise {
+	if c.App.OutgoingOAuthConnections() == nil || c.App.License() == nil || !model.MinimumEnterpriseLicense(c.App.License()) {
```

Comment:
nit: We are already checking for license `nil` in the new helper function
```suggestion
	if c.App.OutgoingOAuthConnections() == nil || !model.MinimumEnterpriseLicense(c.App.License()) {
```

Reply:
Done.

Reply:
I've fixed the nil checks everywhere.